### PR TITLE
person detection 0301 0303

### PR DESCRIPTION
**Pull request by morkovka1337**
add new pre-trained models, in general, they provide better detection accuracy, comparing with previous generation (person-detection-02xx)

- [x] person-detection-0301
- [x] person-detection-0302
- [x] person-detection-0303

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
@morkovka1337 please address RuntimeError: models/intel/device_support.md: "person-detection-0301" model reference is missing (add model to the table)

**Comment by morkovka1337** [comment]
Note that model complexity for the 0301 model is inaccurate right now in the readme due to bug in the model analyzer.

File: models/intel/device_support.md [hunk 1]
```diff
@@ -72,6 +72,9 @@
 | person-detection-0201 | YES | YES | YES |
 | person-detection-0202 | YES | YES | YES |
 | person-detection-0203 | YES | YES | YES |
+| person-detection-0301 | YES | YES | YES |
```
Comment (vladimir-dudnik) [inline models/intel/device_support.md:75]: have you really tested on GPU and MyriadX ? If not please not put `YES`, we will update based on accuracy validation later. Notice, working on CPU is a must (one of the release criteria)

Comment (morkovka1337) [reply models/intel/device_support.md:75]: No, did not test. Ok, I will put `NO`.

Comment (vladimir-dudnik) [reply models/intel/device_support.md:75]: leave just empty cell in the table. We do not put NO explicitly

Comment (morkovka1337) [reply models/intel/device_support.md]: Removed untested devices.

**Review by vladimir-dudnik — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Pull request merged**